.github/workflows/crypto3-testing-linux.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: nix flake -L check .?submodules=1#
         env:
           NIX_CONFIG: |
-            cores = 2
+            cores = 8
             max-jobs = 1
```

Comment:
Try to set max-jobs to 2 and cores to 4 (or even 2 2). It could decrease testing time

Reply:
Much better. 55 min -> 20 min